### DISCUSSION TITLE
Add an Alternative for Cards with a Round Slider

### DISCUSSION THREAD

**Discussion by TheFrozenDuck** [home-assistant/frontend#17428] [Dashboard]
The round slider which is used, e.g. in the light cards, is in some instances not ideal in terms of accessibility. For example, the round slider:

- cannot be changed with the arrow keys, unlike the vertical sliders in the detail page, (Web UI on a desktop)
- dragging the slider from left to right results in big jumps,  (Web UI on a desktop)
- and if the slider loses focus for whatever reason on the phone, the page scrolls up.  (Android app)

It would be awesome if by default one could choose between a card using the round slider or a horizontal slider.

**Comment by Xitee1** [comment]
You can use custom cards for this, for example [Mushroom](https://github.com/piitaya/lovelace-mushroom) which has lots of beautiful cards. If that's not exactly what you want, just search for other third-party cards which satisfy you. If you haven't already, I suggest you to install the community store HACS for the installation of custom cards.
That would be the easiest solution until HA includes a card like you explained by default.
